runtime/Builtins/Array.h

```diff
@@ -347,6 +347,10 @@ class ArraySlice {
     {
         VERIFY(m_storage);
         VERIFY(m_offset < m_storage->size());
+
+        if(m_offset + m_size >= m_storage->size()) {
+            m_size = m_storage->size() - m_offset;
+        }
```

Comment:
Do we not have a clang-format check in CI?

This should be
```suggestion
        if (m_offset + m_size >= m_storage->size())
            m_size = m_storage->size() - m_offset;
```

Reply:
Not currently in CI. I have updated to remove the braces/ fix the spacing.